lib/src/FauiLocalStorage.dart

```diff
@@ -34,46 +17,68 @@ class FauiLocalStorage {
     print("sso: deleted locally");
   }
 
-  static Future trySignInSilently(String apiKey) async {
+  static trySignInSilently(String apiKey) async {
     print("sso: started silent sign-in");
     try {
-      await _initialize();
-      FauiUtil.throwIfNullOrEmpty(value: apiKey, name: "apiKey");
-      String v = _getLocalValue(_LocalKey);
-      if (v == null || v == "null") {
-        print("sso: no user stored");
-        return;
-      }
+        await _getLocalValue(_LocalKey).then((String v) async {
```

Comment:
If you use the keyword 'await', you do not need 'then'.
Next statement will wait for this statement to finish.
There are two styles to manage asynchrony: (1) with 'then' and (2, more modern) with async/await/Future.
In Faui I use the second way. 

See more details here:  https://dart.dev/codelabs/async-await.

Can you remove 'then', please?